### ISSUE TITLE
ModuleNotFoundError when importing bioscrape from inside bioscrape directory.

### ISSUE THREAD

**Issue by ayush9pandey** [biocircuits/bioscrape#21]
MWE to produce this error:

```
>> cd bioscrape/
>> python
>> import bioscrape
'Module not found error bioscrape.random not found'
(this is an error in __init__.py)
```

**Comment by ayush9pandey** [comment]
As this error only occurs when importing from the root directory (which no one should do, ideally), but people end up creating their files in this root directory sometimes and end up with this error.  If this was a pure python package, this error wouldn't exist but since bioscrape imports work from the compiled version only (i.e. the installed package in sites-packages). 

What could be a good way to solve this issue? One way is to simply issue a warning or an error if someone tries to import bioscrape from the root directory (they should not be doing this). The other way is to some how force all the imports to look at sites-packages by default. 

Any ideas @sclamons @WilliamIX ?

**Comment by WilliamIX** [comment]
@ayush9pandey I think an error or warning would be fine, maybe in __init__ of the package?

**Comment by ayush9pandey** [comment]
Yes, but how do we detect that the user is inside the directory? There must be a way tho, I didn't look closely for a solution. But if it can be done, we should do it

**Comment by ayush9pandey** [comment]
Adding the package to `src/` will solve this issue. Most other python packages do this, and it is probably the correct packaging practice.

**Comment by ayush9pandey** [comment]
Fixed in `documentation` #165
